oauth2_http/java/com/google/auth/oauth2/JwtClaims.java

```diff
@@ -66,12 +71,13 @@
   /**
    * Returns additional claims for this object. The returned map is not guaranteed to be mutable.
    *
-   * @return additional claims
+   * @return additional string claims
```

Comment:
I'm confused here. The type no longer says it's a string but the doc comment says it is?

Reply:
I think it got back during a rebase. Removed again